<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wgt_hotel_manager</class>
 <widget class="QWidget" name="Wgt_hotel_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>70</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_hotelname">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_manager_or_check">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_room_or_look_room">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>管理房间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_info_or_look_info">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>修改酒店信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
